trunk/src/app/srs_app_rtc_source.hpp

```diff
@@ -706,6 +714,19 @@ class SrsRtcSendTrack
     virtual srs_error_t on_rtp(SrsRtpPacket* pkt) = 0;
     virtual srs_error_t on_rtcp(SrsRtpPacket* pkt) = 0;
     virtual srs_error_t on_recv_nack(const std::vector<uint16_t>& lost_seqs);
+public:
+    srs_error_t send_rtcp_sr(int64_t now_ms);
+    void update_rtp_static(int64_t len, uint32_t rtp_ts);
+public:
+    srs_error_t handle_rtcp_rr(const SrsRtcpRB& rb, int64_t now_ms);
+protected:
```

Comment:
Directly assigning values to member variables in a C++ class seems to be a syntax introduced in C++11.